terratest/test/helpers.go

```diff
@@ -185,3 +186,39 @@ func assertGslbDeleted(t *testing.T, options *k8s.KubectlOptions, gslbName strin
 
 	assert.Equal(t, deletionExpected, deletionActual)
 }
+
+
+// WaitForLocalGSLB waits until GSLB is ready
+func WaitForLocalGSLB(t *testing.T, host string, port int, expectedResult []string) (output []string, err error) {
+	const maxRetries = 100
+	const sleepBetweenRetries = time.Second * 1
+	var actionDescription = fmt.Sprintf("%s:%v", host, port)
+
+	if len(expectedResult) == 0 {
+		return
+	}
+	sort.Strings(expectedResult)
+	for i := 0; i <= maxRetries; i++ {
```

Comment:
Conceptually and by implementation it is very close to existing https://github.com/AbsaOSS/k8gb/blob/2d1b7a6607631d1a75e4b6ff8d8a02c6d7feba8e/terratest/test/helpers.go#L68 . Is there any reason we are not reusing it?